default_payment_config.xml

```diff
@@ -17,4 +17,26 @@
         <wdoxidee_descriptor>1</wdoxidee_descriptor>
         <wdoxidee_additional_info>1</wdoxidee_additional_info>
     </payment>
+    <payment>
+        <oxid>wdcreditcard</oxid>
+        <oxactive>0</oxactive>
+        <oxtoamount>1000000</oxtoamount>
+        <oxdesc>Wirecard Kreditkarte</oxdesc>
+        <oxdesc_1>Wirecard Credit Card</oxdesc_1>
+        <wdoxidee_logo>creditcard.png</wdoxidee_logo>
+        <wdoxidee_transactionaction>purchase</wdoxidee_transactionaction>
```

Comment:
didn't we change that to `pay` and `reserve` already?

Reply:
changed to pay after rebase